tests/baselines/reference/newLineFlagWithCRLF.js

```diff
@@ -1,9 +1,9 @@
-//// [newLineFlagWithCRLF.ts]
+//// [newLineFlagWithCRLF.ts]
 var x=1;
```

Comment:
sorry missed the change in name of the variable.. can you also make sure that these files are never normalized.. or if you are deleting the code, then we are fine :)